src/service/performance-impl.js

```diff
@@ -715,30 +715,27 @@ export class Performance {
    *     this directly.
    */
   tick(label, opt_delta) {
-    const value = opt_delta == undefined ? this.win.Date.now() : undefined;
+    let data;
+    let storedVal;
+
+    // Absolute value case (not delta).
+    if (typeof opt_delta === 'undefined' || opt_delta === null) {
```

Comment:
`typeof` isn't necessary for local variables, it'll never throw an error. We can just do `opt_delta == null`

Reply:
do you mean `opt_delta == undefined`?